### ISSUE TITLE
Sass not compiling

### ISSUE THREAD

**Issue by krisawzm** [octobercms/october#1014]
I can't seem to get Sass compiling.
Markup used:
```html
<link rel="stylesheet" href="{{ [
	'assets/sass/main.scss']|theme }}">
```

This creates a /combine-url containing the raw Sass syntax. It also combines multiple .scss-files if specified. It seems the only issue is with compiling Sass into CSS - everything else seems ok.

Compiling Less, however, works just fine using the same syntax. Compiling Sass should work the same way, right?

October build: 230
Running Laravel Homestead (on Yosemite) with all default settings.

tl;dr: Sass combines, then takes a nap when it's time to compile.

**Comment by scottbedard** [comment]
To my knowledge October only supports LESS files, not SASS or SCSS.

**Comment by krisawzm** [comment]
It seems you're right. Hopefully it will be implemented at some point, as meny people would rather use Sass over Less.

**Comment by scottbedard** [comment]
Possibly, I wouldn't bet on it though, everything else with October uses LESS. I would just set up Koala or some other compiler on your own to leverage SASS.

**Comment by NemoD503** [comment]
+1 for @scottbedard, You also can use grunt or gulp with "watch" function for compiling. it work very quickly, each time, when you change scss file

**Comment by Eein** [comment]
Sorry to necro this issue, but i'm curious is this is still in the pipeline. 

I wouldn't mind working on a pull request to add this functionality for those who need it. It looks like theres a bit of work on adding scss/sass to the CMS, but simply set aside for now. 

@daftspunk It would be nice if you could add your opinion if this is something you want to even be in the framework.

Thanks!

**Comment by peederspad** [comment]
+1 for SASS/SCSS Support :)

**Comment by krisawzm** [comment]
Okey, so after using OctoberCMS for a while, my opinion is it's not necessary to support this. In most cases, you will be better off using a proper task runner such as Grunt or Gulp, or even a GUI such as Koala. With a proper task runner you can decide for yourself which CSS preprocessor to use. And you will also be able to do other things, such as using autoprefixer, optimizing images, watching for changes, plus all the other things a task runner can do.

I honestly don't even think the Less compiler is necessary. Yes, it's kinda cool to have it built in, but in reality it's really just slow. Plus you have to empty the cache every time you make changes, which is annoying.

**Comment by flip4bytes** [comment]
@krisawzm +1

I think this is another feature that is just kinda unnecessary. I personally use Stylus, some use Less, and some use Sass/Scss. It's kinda against the open source idea of forcing people to use Less if they want that functionality. Either support none, or support them all. I'm just going to keep using Gulp locally and monitoring my changes within my git repo. Not really sure how often you should be making these changes from the site on the server anyway.

**Comment by riuson** [comment]
krisawzm, flip4bytes  **-2**
The functionality has already been implemented. If you do not like it - do not use. It is open source.

**Comment by JumpLink** [comment]
+1
[There are any plans to support SASS?](https://octobercms.com/forum/post/how-compile-sass-files-with-octobercms?page=1#post-14548)

**Comment by daftspunk** [comment]
We would more than likely implement this officially when Bootstrap 4 is launched, so we can support it. For now there is a plugin that can give you this functionality, check it out!

https://octobercms.com/plugin/willing-scss

**Comment by daslicht** [comment]
the rest of the world is moving away from Less...

**Comment by JumpLink** [comment]
Thank you, I have not found this plugin previously

**Comment by amdad** [comment]
Unfortunately, this plugin not working at all in new version. Didn't even install needed files. LESS is less. Scss is MORE... and better. 

Look at this statistics from Codepen: http://codepen.io/chriscoyier/pen/wMVLjP

% OF PENS USING CSS PREPROCESSOR:

**SCSS 72.20%**
LESS: 15.81%
SASS: 8.10%

I really don't care about bootstrap. It is great tool for community driven backend framework (because of it's popularity). Personally I will never use it on frontend. 

I thought, compiler in October is not just for such a basic job as compiling October backend theme or demo frontend themes. Backend could live great even without this. October allows freedom on frontend. And I will not use today abandoned/obsolete technology. Not want rewrite less to scss in future. Also don't want external compilers to do such a basic jobs. 

Compiler is rather as a developer tool. There is no link between bootstrap and compiler. You can add scss support because it is needed for frontend. And as you know it is best choice for today and for future. Looking forward another big thing is PostCSS. Even Bootstrap authors said v5 will be probably PostCSS and is not now because already bootstrap was rewritten to scss.

So perfectly would be to have SCSS and PostCSS (as preprocessor) support now. If not both - SCSS.

Sticking with libraries that is nishe in global project is bad. I remember situation when Joomla for years stick with Mootools instead of jQuery. Of course finally switch.  Mootools was great library long time ago, but with time lack of interest of global community in favour of jQuery causing results mostly   limitations.  I know lack of complier is not a big deal here, But if you have one let it be best one. 

You can add them today without any problem because it will be used from this day for many years ahead. LESS will die, then it will become even bigger problem. Because if many themes will be based on this you'll need o keep it in October for some time to not break them. Faster switch to better technology of future = _LESS_ PROBLEMS in future.

**Comment by JumpLink** [comment]
@amdad christianWilling ist currently working on to integreate scss support to the october core, see https://github.com/octobercms/library/pull/225

**Comment by amdad** [comment]
That's cool! What about things like Compass? Is there will be some easy way to include them into compiling process? Power of LESS/SCSS is in great tools like this one.
